linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/oval/shared.xml

```diff
@@ -0,0 +1,44 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("All password change date is in the past.") }}}
+    <criteria>
+      <criterion comment="passward last change was in the past" test_ref="test_accounts_password_last_change_time_secs"/>
+    </criteria>
+  </definition>
+
+  <local_variable id="var_accounts_password_last_change_time_secs" datatype="int" version="1" comment="last change component of password entry">
+    <arithmetic arithmetic_operation="multiply">
+        <object_component object_ref="object_accounts_password_all_chage_in_past" item_field="chg_lst"/>
+      <literal_component datatype="int">86400</literal_component>
+      </arithmetic>
+  </local_variable>
+
+  <local_variable id="var_accounts_password_last_change_time_diff" datatype="int" version="1" comment="last change component of password entry compared to current time">
+    <time_difference format_2="seconds_since_epoch">
+        <variable_component var_ref="var_accounts_password_last_change_time_secs"/>
+    </time_difference>
+  </local_variable>
+
+  <ind:variable_object  id="object_accounts_password_last_change_time_diff" version="1">
+     <ind:var_ref>var_accounts_password_last_change_time_diff</ind:var_ref>
+   </ind:variable_object>
+
+   <ind:variable_state id="state_accounts_password_last_change_time_diff" version="1">
```

Comment:
This is the error reported by many CI tests:
`oval:ssg-state_accounts_password_last_change_time_diff:ste:1 - a var_check has been supplied for the ind:value entity so a var_ref must also be provided`

Reply:
Dropped that thing in [135d663](https://github.com/ComplianceAsCode/content/pull/10031/commits/135d663c3dac8a18fb51088c0b758f095081c5fd) , probably remained from previous version of how the state section was written. Thanks :bow: